UnitystationLauncher/Constants/ApiUrls.cs

```diff
@@ -2,13 +2,20 @@ namespace UnitystationLauncher.Constants;
 
 public static class ApiUrls
 {
-    public const string ApiBaseUrl = "https://api.unitystation.org";
+    private const string ApiBaseUrl = "https://api.unitystation.org";
     public const string ServerListUrl = $"{ApiBaseUrl}/serverlist";
     public const string ValidateUrl = $"{ApiBaseUrl}/validatehubclient";
     public const string ValidateTokenUrl = $"{ApiBaseUrl}/validatetoken?data=";
     public const string SignOutUrl = $"{ApiBaseUrl}/signout?data=";
 
-    public const string ChangelogBaseUrl = "https://changelog.unitystation.org";
+    private const string ChangelogBaseUrl = "https://changelog.unitystation.org";
     public const string Latest10VersionsUrl = $"{ChangelogBaseUrl}/all-changes?format=json&limit=10";
     public const string LatestBlogPosts = $"{ChangelogBaseUrl}/posts/?format=json";
+
+    private const string CdnBaseUrl = "https://unitystationfile.b-cdn.net";
+    public const string GoodFilesBaseUrl = $"{CdnBaseUrl}/GoodFiles";
```

Comment:
Can these be  get only properties instead of public fields?

Reply:
Sure, is this what you had in mind?